main_test.go

```diff
@@ -50,7 +43,7 @@ func TestKeygenInTmpFolder(t *testing.T) {
 
 	assert.Nil(t, err)
 
-	out, err := RunCliWithArgs(t, []string{"gen", "--keys-folder", tmpDir, "--name", "mykey"})
+	out, err := RunCliWithArgs(t, []string{"gen", "--keys-folder", tmpDir, "--key", "mykey"})
```

Comment:
It's a bit weird to name this param "key" given we are generating a key (we're just specifying the file name). Why did you think "--name" was a bad name? Maybe `--file-name` would be better?

Reply:
How about '--key-name' and '-k'?

Reply:
`--key-name / -k` sounds good!